core/basicobject/instance_exec_spec.rb

```diff
@@ -84,17 +84,17 @@ def foo
     end.should raise_error(TypeError)
   end
 
-quarantine! do # Not clean, leaves cvars lying around to break other specs
-  it "scopes class var accesses in the caller when called on an Integer" do
-    # Integer can take instance vars
-    Integer.class_eval "@@__tmp_instance_exec_spec = 1"
-    (defined? @@__tmp_instance_exec_spec).should == nil
-
-    @@__tmp_instance_exec_spec = 2
-    1.instance_exec { @@__tmp_instance_exec_spec }.should == 2
-    Integer.__send__(:remove_class_variable, :@@__tmp_instance_exec_spec)
+  quarantine! do # Not clean, leaves cvars lying around to break other specs
+    it "scopes class var accesses in the caller when called on an Integer" do
+      # Integer can take instance vars
+      Integer.class_eval "@@__tmp_instance_exec_spec = 1"
+      (defined? @@__tmp_instance_exec_spec).should == nil
+
+      @@__tmp_instance_exec_spec = 2
+      1.instance_exec { @@__tmp_instance_exec_spec }.should == 2
+      Integer.__send__(:remove_class_variable, :@@__tmp_instance_exec_spec)
+    end
   end
-end
```

Comment:
I'm not totally sure about these `quarantine!` blocks, they're kind of special, do we want to keep the special indentation for those?

Reply:
Right for `quarantine!` we sometimes avoided to reindent to minimize changes and noise in diffs/git log -p.
I think it would be nice to avoid indenting more under `quarantine!`, but does RuboCop support that?

Reply:
It looks like there is no way to configure that from a quick look at https://docs.rubocop.org/rubocop/cops_layout.html#configurable-attributes-31
It's fine to reindent, we don't quarantine/unquarantine often.

Reply:
There are two ways to work around this:
* Add an exclude for the specific cop for this file, which means we ignore this specific cop in the whole file (not just this quarantine line)
* Add a `rubocop:disable Layout/IndentationConsistency` line in the source to make Rubocop ignore the specific line or block. This means we're mixing up Rubocop with the code

I think both of these options are pretty ugly, so I would be okay with not added any exceptions here.

Reply:
Yeah let's just reindent as you did.
In general I would prefer per-file exclude instead of rubocop comments, but even better if we don't have to exclude at all.